README.md

```diff
@@ -97,11 +103,154 @@ Please see Hazelcast Node.js [code samples](https://github.com/hazelcast/hazelca
 
 You can also refer to Hazelcast Node.js [API Documentation](http://hazelcast.github.io/hazelcast-nodejs-client/api/current/docs/).
 
-# Configuration
+# Configuration Overview
+
+## 1. Configuration Options
```

Comment:
I don't think we need the heading numbers here and in the below headings, they don't look good. TOC will take care of it smoothly.